week_3/2018202150FTC/src/claw.h

```diff
@@ -0,0 +1,65 @@
+#include <regex>
+#include <set>
+#include <vector>
+#include <algorithm>
+#include <iostream>
+#include <fstream>
+#include <queue>
+#include <string>
+#include <string.h>
+#include<sstream>
+#include<string.h>
+ using namespace std;
+
+class StringUtils
```

Comment:
Checkout these standard library functions:

* [`std::atoi`, `std::atol` and `std::atoll`](https://en.cppreference.com/w/cpp/string/byte/atoi)
* [`std::atof`](https://en.cppreference.com/w/cpp/string/byte/atof)